### ISSUE TITLE
Please provide "release" docker tag or similar

### ISSUE THREAD

**Issue by fsateler** [certbot/certbot#2894]
It would be great to have a docker tag that tracks the releases. Currently one needs to either track git master, or use versioned tags, and change on each release.

**Comment by stephank** [comment]
We actually rely on Docker images in production, and the official images worked fine, even though they're on master. Downside is mostly that we get debug logging no matter `-q`.

There's a bunch of other images on Docker Hub, but they seemed to have quite complicated Dockerfiles. Same actually for the official Dockerfiles, but maybe they serve a different purpose.

So I set out to build a simple image that mimics what `certbot-auto` creates in a virtualenv as close as possible:

https://hub.docker.com/r/angrybytes/certbot/~/dockerfile/

Only interesting thing I encountered is that the current [letsencrypt-auto-requirements.txt](https://github.com/certbot/certbot/blob/master/letsencrypt-auto-source/pieces/letsencrypt-auto-requirements.txt) actually doesn't match what's in the current `certbot-auto`. I'll try to track what's in actual `certbot-auto` releases.

Other than that, biggest thing here is probably just settings up build infrastructure. (I'm not very familiar with Travis.)
